### ISSUE TITLE
Backups now including media

### ISSUE THREAD

**Issue by andyp05** [lavinir/hassio-onedrive-backup#168]
I have been using the addon for a year and just noticed that the media folder is now included in daily backups.
I checked the settings and `Exclude media folder` is still on and File Sync Paths is still set to: `/media/**/*.*`
These settings haven't changed in months.

Going back through the backups, it looks like it started on Jan 22 when the backup size went from approx. 300MB to 4GB and has stayed up there ever since.
In the /config folder the media directory is empty. The FileSync directory has a media sub-directory with my Frigate recordings under that.

![image](https://github.com/lavinir/hassio-onedrive-backup/assets/14006401/040009dd-e374-473a-badf-b77578c2c306)

If I open the backup file it shows:
![image](https://github.com/lavinir/hassio-onedrive-backup/assets/14006401/0020dff6-c136-44bd-b448-e7be1cff759f)

The backups are working otherwise and sent to OneDrive, however they are huge.

Don't know if this is related, but in the logs, there are many warnings, but the backups look fine.

```
01/29/2024 22:08:36 [33] Info: Reached Max Online Backups (10)
01/29/2024 22:08:36 [33] Info: Found 1 backups to delete from OneDrive.
01/29/2024 22:08:36 [33] Info: Deleting item: ha_backup_2024-01-21-14-02..tar
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : blueprints
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : custom_components
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : deps
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : media
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : packages
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : themes
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : tts
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : www
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : blueprints
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : custom_components
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : deps
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : media
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : packages
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : themes
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : tts
01/29/2024 22:08:37 [20] Warning: Unrecognized file found in backup folder : www
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : blueprints
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : custom_components
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : deps
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : media
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : packages
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : themes
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : tts
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : www
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : blueprints
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : custom_components
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : deps
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : media
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : packages
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : themes
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : tts
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : www
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : blueprints
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : custom_components
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : deps
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : media
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : packages
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : themes
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : tts
01/29/2024 22:13:36 [14] Warning: Unrecognized file found in backup folder : www
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : blueprints
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : custom_components
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : deps
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : media
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : packages
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : themes
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : tts
01/29/2024 22:13:37 [27] Warning: Unrecognized file found in backup folder : www
```
Let me know if I can get you any more info to track this down.

Thanks

**Comment by lavinir** [comment]
Something is not right there.
First - Notice the Media folder in your backup is 0 bytes. It is not the cause of the large backup size. The cause is the "FileSync" folder that is part of your Home Assistant backup. What is the 'FileSync' folder you have under your Configuration dir ?

Second - Those warnings should not happen - it means these files exist in the backup folder in OneDrive. Have you manually extracted / moved files there ?

**Comment by andyp05** [comment]
I did not do anything earlier this month except maybe a HA upgrade. The Frigate addon is the only thing that stores media. I have not changed that setup or upgraded the addon in months. 
I did not create the FileSync folder. I just noticed it is there. I thought maybe this plugin created it given the name. 
I have not manually done anything with the backups. No restore, extraction, or manual deletions. 
I searched for the word 'FileSync' in the config folder and no instances were found.

**Comment by andyp05** [comment]
I deleted the FileSync directory in /config.
Frigate stores its media in /media not in /config/media so that is why it is 0 bytes.

I also noticed that the files and directories in the log were in OneDrive. I deleted them as well. 
I triggered a new backup and it is back to 350MB and no more warnings. I have no idea what happened. It almost looks like the FileSync directory was dragged/dropped into /config by accident since it was not updated in over a week. 
Thanks for getting back to me on this. I will close and if anything happens again, I will reopen.
